packages/expo-updates/e2e/__tests__/fixtures/project_files/e2e/tests/utils/server.ts

```diff
@@ -70,13 +72,13 @@ app.post('/post', (req, res) => {
   }
 });
 
-app.post('/log', (req, res) => {
+app.post('/log', (req: { body: { logEntries: never[]; }; }, res: { set: (arg0: string, arg1: string) => void; send: (arg0: string) => void; }) => {
```

Comment:
Instead of inlining the types here, it might be more clear to define them a line before 🤔